packages/components/file-uploader/src/states/download.js

```diff
@@ -0,0 +1,29 @@
+import { calcFileSize } from '../utils/calcFileSize';
+
+export function downloadState(file) {
+  const fileExtension = file.name.split('.').pop();
+
+  return `
+  <div class="file-card file-card--download">
+    <div class="file-info--wrapper">
+      <div class="icon-wrapper">
+        <div class="icon">
+          <svg xmlns="http://www.w3.org/2000/svg" width="35" height="44" viewBox="3 1 18 22" fill="none" stroke="currentColor" stroke-width="1" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file">
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 220 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ALineLength)